src/components/drawer/drawer.js

```diff
@@ -1,24 +1,24 @@
 import template from "./drawer.html";
 import { MDCPersistentDrawer } from "@material/drawer";
 import { ButtonMoreComponent } from "../button-more/button-more";
+import { AccountsComponent } from "../accounts/accounts-component/accounts-component";
 
 export class DrawerComponent {
-  constructor(mountPoint, props) {
+  constructor(mountPoint, dialogMountPoint, props) {
```

Comment:
you do not need to pass any mount points except -- one first `mountPoint` for component itself. You can mount dialog anywhere in the app, in any component -- it does not hurt its rendering

Reply:
Actually you should move dialog mounting to accounts component

Reply:
@ihor7016 How should we find mount point for inner component? We pass dialog mount point because it corresponds to app.html. If I remove passing by constructor - we have to find this element with document.querySelector , which is also bad